conda_smithy/ci_register.py

```diff
@@ -149,7 +162,10 @@ def add_project_to_travis(user, project):
 
         if count > 5:
             print('  * Unable to register the repo on Travis (is it down?)')
-            return
+            print("    Please also ensure that the user/org name is spelled correctly "
+                  "(case sensitiv).")
+
+            sys.exit(1)
```

Comment:
BTW: should the cli commands (e.g. all `__call__`) return or use `sys.exit(1)` if an error occurred?